wps-cloudformation-template.yaml

```diff
@@ -108,6 +108,9 @@ Parameters:
   geonetworkLayerSearchField:
     Type: String
     Default: 'layer'
+  language:
```

Comment:
Where is this parameter getting used?

Reply:
Whoops - I'll remove - originally started to add this as an environbment parameter but decide that wasn't appropriate

Reply:
Removed